### DISCUSSION TITLE
Screencasts sobre Front-End

### DISCUSSION THREAD

**Discussion by felipeorlando** [frontendbr/forum#566] [Discussão]
Galera, mandem links de screencasts (canais Youtube, sites...) que falem de desenvolvimento Front-End. **Gratuitos** ou pagos. 📹  💻  🎥 

O que vocês acham de criarmos um repositório de aprendizado em Front? Como exemplo, temos [este](https://github.com/felipeorlando/ror-resources) (que eu criei rs) sobre Ruby. 😃

**Comment by woliveiras** [comment]
O canal do Training Center ainda não está com muitos vídeos (só um do @fdaciuk sobre React), mas logo logo teremos muito conteúdo: [Training Center](https://www.youtube.com/channel/UC8reCbWAXloUaFmHhsOh2Xw).

OBS: Quem quiser nos ajudar a encher esse canal de conteúdo, é só entrar em contato comigo pelo Telegram: `@woliveiras`

**Comment by theandersonn** [comment]
@felipeorlando O repositório que vc criou está bem organizado e com bastante informação. Parabéns! 
Um focado em front seria muito bom!

**Comment by matheusml** [comment]
Oi @felipeorlando,

eu fiz dois cursos sobre React lá no [JSCasts](http://jscasts.teachable.com/).
Ambos são inteiramente gratuitos.

Abraços!

**Comment by felipeorlando** [comment]
@woliveiras tc ❤️ 

@theandersonn vlw! é uma ótima ideia, ein! bora lá! 😄 

@matheusml haha fui introduzido ao react pelo seu curso! Foi o ponta-pé inicial. 😃

**Comment by woliveiras** [comment]
Somos dois, @felipeorlando 

Eu curti muito o curso do @matheusml!

**Comment by phenriqueleao** [comment]
Desculpem a desenterrada de tópico, mas é que vi a lista feita pelo @felipeorlando sobre Ruby e não sei se já criaram algo assim (ou pelo menos parecido) sobre front-end.

Se ainda não, apoio fortemente a iniciativa.

**Comment by phenriqueleao** [comment]
Muito legal! Obrigado, @kvnol !

É que eu achei esse muito bem organizado e principalmente centralizado, pensei que pudesse ter algo parecido. Valeu pelos links!

**Comment by kvnol** [comment]
@phenriqueleao Sim! Dei uma pesquisada e como não achei nada, criei o repositório: https://github.com/kvnol/aprenda-frontend e vou utilizar o do @felipeorlando como base. :D

**Comment by felipeorlando** [comment]
@kvnol boa! mas num esquece dos créditos, poxa haha ❤️ 

eu tô sem tempo, mas provavelmente vou voltar a ativa no fds e dar um start inicial no https://github.com/felipeorlando/be-a-frontend 🎉

**Comment by kvnol** [comment]
Massa, @felipeorlando. Créditos adicionados! :heart:

**Comment by didiraja** [comment]
O [CSS Cast Show](https://www.youtube.com/channel/UC9d_htYmYYFUXB2vBKx4NgA) tá começando mas já tem bastante coisa e de qualidade. Fizeram uma cobertura foda do Front-End Week.

**Comment by kvnol** [comment]
O trampo do @afonsopacifer do CSS Cast Show é lindo demais! :heart:

**Comment by afonsopacifer** [comment]
❤️ !!! Ainda tem uns videos da Front Week p sair ;)
@diogomoretti
